src/server.py

```diff
@@ -34,22 +38,29 @@ def server():
                 message_complete = True
 
         try:
-            parse_request(recv_message.decode('utf-8'))
+            client_uri = parse_request(recv_message.decode('utf-8'))
         except TypeError as msg:
             reply = response_error(405, str(msg))
         except ValueError as msg:
             reply = response_error(400, str(msg))
         else:
-            reply = response_ok()
-
+            try:
+                resolved = resolve_uri(client_uri)
+            except IOError:
+                reply = response_error(404, 'file not found')
+            else:
+                # root_dir = os.path.join(PWD, ROOT)
+                reply = response_ok(resolved)
```

Comment:
Looks like, per the `response_ok` method, the variable `resolved` here should be a tuple, although it looks like `response_ok` is expecting just the content body.